local_deployment.py

```
@@ -69,8 +71,7 @@ def run_local_deployment(data):
         print("Deploying Kantra latest")
         if not arg_dependency_file:
             full_zip_name = get_zip_name()
-            url = get_latest_upstream_dependency('konveyor', 'kantra', full_zip_name)
+            image = get_latest_upstream_dependency('konveyor', 'kantra', full_zip_name)
             logging.info(f"Downloading dependencies zip for upstream")
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)